BedrockNode.cpp

```diff
@@ -135,19 +143,29 @@ bool BedrockNode::_peekCommand(SQLite& db, Command* command) {
     return true;
 }
 
-void BedrockNode::_processCommand(SQLite& db, Command* command) {
+void BedrockNode::_setState(SQLCState state) {
+    _dbReady = false;
+    SQLiteNode::_setState(state);
+}
+
+bool BedrockNode::dbReady() {
+    return _dbReady;
```

Comment:
Please rename `_dbReady` to `_upgradeComplete` to make its function/meaning more obvious.